.gitignore

```diff
@@ -46,3 +46,7 @@ rpm/src/
 
 *.pyc
 test/acceptance/properties.json
+cmake_install.cmake
+CMakeFiles
+Makefile
```

Comment:
I think my clone has been compromised.
I have Makefiles and  CMakeFiles  under src/ ...

After removing some of them that I clearly see are produced files I can no longer compile ...

I need to fix my repo and these changes to .gitignore can probably be removed